x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccountsTests.java

```diff
@@ -46,6 +145,280 @@
 
 public class ElasticServiceAccountsTests extends ESTestCase {
 
+    public void testKibanaSystemPrivileges() {
```

Comment:
Not certain we need to replicate ( read: copy) all of the tests here as it uses the same roledescriptor as the kibana_system and we test this in ReservedRolesStoreTests